actionpack/lib/action_controller/metal/implicit_render.rb

```diff
@@ -6,15 +6,40 @@ def send_action(method, *args)
       ret
     end
 
+    # Renders the template corresponding to the controller action, if it exists.
+    # The action name, format, and variant are all taken into account.
+    # For example, the "new" action with an HTML format and variant "phone" 
+    # would try to render the <tt>new.html+phone.erb</tt> template.
+    #
+    # If no template is found +default_missing_template_action+ is called, unless
+    # a block is passed. In that case, it will override +default_missing_template_action+:
+    #
+    #   default_render do
+    #     head 404 # No template was found
+    #   end
+    #
+    # +args+:: passed through to render
```

Comment:
That double colon isn't standard rdoc iirc. I also don't think it communicates very much, which is why I left it out originally.